packages/sidecar/package.json

```diff
@@ -40,6 +40,9 @@
     "typescript": "^5.0.2",
     "vite": "^4.5.3"
   },
+  "optionalDependencies": {
+    "@sentry/node": "^8.14.0"
+  },
```

Comment:
why do we need this? I don't see `@sentry/node` being used anywhere

Reply:
Oh we don't. Leftover from earlier iterations. Great catch, thanks!